docker/README.md

```diff
@@ -30,8 +30,8 @@ Building image and running tests using github actions
 - kafka-url - This is the url to download kafka tarball from. For example kafka tarball url from (https://archive.apache.org/dist/kafka). For building RC image this will be an RC tarball url.
```

Comment:
Could you also update this link to markdown pattern 

`(https://archive.apache.org/dist/kafka)` -> `[Kafka archive](https://archive.apache.org/dist/kafka)`